tools/clang/test/CodeGenDXIL/hlsl/objects/HitObject/hitobject_make.hlsl

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: dxil-1-9
```

Comment:
I don't think this is needed. This is a compiler test not a validator test. It is never run without the latest validator.

Reply:
Deleted that line again